.gitignore

```diff
@@ -13,3 +13,4 @@ cmake_install.cmake
 *.gch
 .idea
 .tool-versions
+Cargo.lock
```

Comment:
The Cargo.lock file shouldn't be gitignored or removed.
It pins our dependencies.

Since your Cargo.toml changes added a new member to the workspace. The updates to Cargo.lock should be part of this PR

Reply:
Done